toString.js

```diff
@@ -0,0 +1,25 @@
+const MajorColors = [
+  "white",
+  "red",
+  "black",
+  "yellow",
+  "violet"
+];
+const MinorColors = [
+  "blue",
+  "orange",
+  "green",
+  "brown",
+  "slate"
+]; 
+function toString(){
+  console.log("manual color coder");

```

Comment:
toString needs to return a string with the reference of all color pairs, not just print it on the console.